src/main/java/org/wildfly/security/util/Alphabet.java

```diff
@@ -269,4 +269,42 @@ public int decode(final int codePoint) {
             }
         };
     }
+    /**
+     * The alphabet used by PicketBox project base 64 encoding.
+     * {@code 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz./}
+     */
+    @Deprecated
```

Comment:
It's probably not necessary to deprecate this, unless we're actually planning on removing it.